README.md

```diff
@@ -1,26 +1,23 @@
 ![.NET on AWS Banner](./logo.png ".NET on AWS")
 
 # AWS .NET Distributed Cache Provider
-AWS Dotnet Distributed Cache Provider provides an implementation of the ASP.NET Core interface [IDistributedCache](https://docs.microsoft.com/en-us/aspnet/core/performance/caching/distributed) backed by AWS DynamoDB. The IDistributedCache interface is used in ASP.NET Core applications to provide a distributed cache store as well as the backing store ASP.NET Core session state.
-
+The AWS .NET Distributed Cache Provider provides an implementation of the ASP.NET Core interface [IDistributedCache](https://docs.microsoft.com/en-us/aspnet/core/performance/caching/distributed) backed by Amazon DynamoDB. An `IDistributedCache` implementation may used in ASP.NET Core applications to store session state data.
 
 # Getting Started
-.NET has an interface for distributed caching called [IDistributedCache](https://docs.microsoft.com/en-us/aspnet/core/performance/caching/distributed). This library provides an implementation that uses AWS DynamoDB as the underlying datastore.
-
-.NET's convention is to use Dependency Injection to provide services to different aspects of an application's logic. This library provides extensions to assist the user in injecting this implementation of IDistributedCache as a service for other services to consume. 
+.NET specifies an interface for distributed caching called [`IDistributedCache`](https://docs.microsoft.com/en-us/aspnet/core/performance/caching/distributed). This library provides an implementation that uses Amazon DynamoDB as the underlying data store.
 
-This library also provides some configuration options to help fit this library into your application.
+.NET's uses [dependency injection](https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection) to provide services to different objects that rely on them. This library provides extensions to assist the user in injecting this implementation of `IDistributedCache` as a service for other objects to consume.
```

Comment:
nit: change ".NET's uses" to ".NET uses"

Reply:
Done.